static/js/invite.js

```diff
@@ -208,6 +213,7 @@ exports.initialize = function () {
     $("#invite-user").on("change", "#generate_multiuse_invite_radio", () => {
         $("#invitee_emails").prop("disabled", false);
         $("#submit-invitation").text(i18n.t("Invite"));
+        document.querySelector("#copy_link").type = "hidden";
```

Comment:
Can you use jQuery for these manipulations, for consistently with the rest of the codebase?

Reply:
yeah sure